File: app/src/main/java/net/veldor/flibustaloader/utils/TransportUtils.java
package net.veldor.flibustaloader.utils;

import android.content.Intent;
import android.content.pm.PackageManager;

import net.veldor.flibustaloader.App;

public class TransportUtils {
    public static boolean intentCanBeHandled(Intent intent){
        PackageManager packageManager = App.getInstance().getPackageManager();
        return intent.resolveActivity(packageManager) != null;
    }
}

File: app/src/main/java/net/veldor/flibustaloader/utils/TransportUtils.kt
package net.veldor.flibustaloader.utils

import net.veldor.flibustaloader.App
import android.content.Intent

object TransportUtils {
    @kotlin.jvm.JvmStatic
    fun intentCanBeHandled(intent: Intent): Boolean {
        val packageManager = App.instance.packageManager
        return intent.resolveActivity(packageManager) != null
    }
}